client/src/components/Home.js

```diff
@@ -49,11 +49,6 @@ const Home = ({ user, logout }) => {
     setConversations((prev) => prev.filter((convo) => convo.id));
   };
 
-  const saveMessage = async (body) => {
-    const { data } = await axios.post('/api/messages', body);
-    return data;
-  };
-
```

Comment:
This change is not required to solve `the async/await bug`; please revert the changes you made here to the original implementation. :slightly_smiling_face: